env/WCOSS2.env

```
@@ -86,40 +91,53 @@ elif [ $step = "gldas" ]; then
 
 elif [ $step = "eobs" ]; then
 
+    export OMP_PLACES=cores
+    export OMP_NUM_THREADS=$nth_eobs
+    export OMP_STACKSIZE=1G
+    export FI_OFI_RXM_SAR_LIMIT=3145728
+
     nth_max=$(($npe_node_max / $npe_node_eobs))
 
     export NTHREADS_GSI=${nth_eobs:-$nth_max}
     [[ $NTHREADS_GSI -gt $nth_max ]] && export NTHREADS_GSI=$nth_max
-    export APRUN_GSI="$launcher -n ${npe_gsi:-${npe_eobs:-$PBS_NP}} -ppn $npe_node_eobs"
+    export APRUN_GSI="$launcher -n ${npe_gsi:-${npe_eobs:-$PBS_NP}} -ppn $npe_node_eobs --cpu-bind core --depth $nth_eobs"
```

I think this should be:
```suggestion
    export APRUN_GSI="$launcher -n ${npe_gsi:-${npe_eobs:-$PBS_NP}} -ppn $npe_node_eobs --cpu-bind core --depth $NTHREADS_GSI"
```